database/staff_shifts.sql

```diff
@@ -0,0 +1,13 @@
+create table staff_shifts (
+      port		  text	  NOT NULL,
+      terminal    text	  NOT NULL,
+      shift_name  text 	  NOT NULL,
+      start_date  date   NOT NULL,
+      start_time  text    NOT NULL,
```

Comment:
Wondering what the reasoning is for storing the date and time as separate fields. I would think it would make things simpler if they're stored as a single timestamp

Reply:
Start date is and start time are bit different . Start date is more of when shift is starting but start time is not relegated to date as it depends on frequency which is default to daily .